zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -25,11 +25,15 @@
       <term name="note">wie Anm.</term>
       <term name="et-al">et al.</term>
       <term name="editor" form="short">Hg.</term>
+      <term name="translator" form="verb-short">übers. von</term>
+      <term name="editortranslator" form="verb-short">hg. und übers. von</term>
```

Comment:
These terms should already be translated in `de-DE` standard locale exactly as you want them: https://github.com/citation-style-language/locales/blob/master/locales-de-DE.xml#L279

Thus, there should be no reason to define them here again, i.e. delete those two lines here.